core/lib/core/observatory/switch.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Core.Observatory.Switch do
+  use Core.Signals.Handlers
+  alias Core.Observatory
+
+  def dispatch(:next_action_created, %{user: user} = message) do
+    Observatory.local_dispatch(:next_action_created, [user.id], message)
```

Comment:
Dit gaan we nog abstracter maken zodra we wat meer  concrete voorbeelden hebben, toch?

Reply:
Ik voorzie hier inderdaad nog wel abstractie mogelijkheden. Het ding is dat voor nu er twee dingen gebeuren:
1. Configuratie van signal naar observables
2. Key selectie; in dit geval user.id
Hoe we dit het beste kunnen generaliseren hangt ook af van wat er met signals gebeurd. Ik verwacht dat die wat meer nodig gaan hebben dan atom + payload.